### PR TITLE
Implement :text cell type

### PR DESCRIPTION
**Pull request by arkadiybutermanov**
Issue: https://github.com/randym/axlsx/issues/348

### CODE REVIEW

**Comment by arkadiybutermanov** [comment]
@randym could you please take a look?

**Comment by randym** [comment]
@arkadiybutermanov travis issues have been resolved. Can I ask you to create specs covering serialization of '=' and '-' prefixed strings so this can be merged.

**Comment by arkadiybutermanov** [comment]
@randym done! Thanks!

File: test/workbook/worksheet/tc_cell.rb [hunk 1]
```diff
@@ -319,6 +319,21 @@ def test_to_xml_string_array_formula
     assert(doc.xpath("//f[@ref='A1']"))
```
Comment (arkadiybutermanov) [inline test/workbook/worksheet/tc_cell.rb]: @randym also, this test seems to be false-positive. `doc.xpath(...)` always returns non-falsey value. It returns empty array if node is not found. That's why I've decided to use `empty?` and `any?` in my test.

File: test/workbook/worksheet/tc_cell.rb [hunk 2]
```diff
@@ -319,6 +319,21 @@ def test_to_xml_string_array_formula
...
+    ws = p.workbook.add_worksheet do |sheet|
+      sheet.add_row ["=1+1", "-1+1"], type: :text
+    end
+    doc = Nokogiri::XML(ws.to_xml_string)
+    doc.remove_namespaces!
+
+    assert(doc.xpath("//f[text()='1+1']").empty?)
+    assert(doc.xpath("//t[text()='=1+1']").any?)
+
+    assert(doc.xpath("//f[text()='1+1']").empty?)
```
Comment (randym) [inline test/workbook/worksheet/tc_cell.rb]: This is the same assertion as line 330?

File: test/workbook/worksheet/tc_cell.rb (hunk 1 above)
Comment (randym) [inline test/workbook/worksheet/tc_cell.rb]: good catch - I'm sure there are others like this that can be cleaned up as we go.

**Comment by randym** [comment]
thanks @arkadiybutermanov

**Pull request merged**